.gitignore

```diff
@@ -5,3 +5,5 @@ build
 __pycache__
 .eggs/
 pytrainer.egg-info
+Notes
+Todo
```

Comment:
I'm not going to merge this commit. If you want to keep personal files in the repo and ignore them please use .git/info/exclude instead.

Reply:
Done. Submitted a new PR (#238) that splits the commit into 2. You can ignore/cancel this one.